### PR TITLE
Fix #2749 and #2488: Manage Controlled Processes does not show any process

### LINKED ISSUE #2749: Manage Controlled Processes does not show any process
## Required Information

* [x] AEM Version, including Service Packs, Cumulative Fix Packs, etc: 6.5.6.0
* [x] ACS AEM Commons Version: 4.12.0
* [ ] Reproducible on Latest? yes/no

### Expected Behavior

Shows a list of executed process

### Actual Behavior

Does not show any process

### Steps to Reproduce
Go to /apps/acs-commons/content/manage-controlled-processes.html
This is currently only happenning on PROD and the only way to see the list is accesing the instance through Ip

### Pictures

linkview2
ipView2

### PR DESCRIPTION
**Pull request by YegorKozlov**
Fix #2749 and #2488:

When serializing data using

```
gson.toJson(object, writer);
```
we need to ensure response.getWriter() is closed in the end, otherwise it might return an empty json. 

see the explanation in https://github.com/Adobe-Consulting-Services/acs-aem-commons/issues/2749#issuecomment-986244291

### CODE REVIEW

**Comment by kwin** [comment]
The servlet's response is automatically closed by the servlet engine. Closing explicitly might cause issues with filters https://stackoverflow.com/a/1159244. The issue must be a different one...

**Comment by kwin** [comment]
Usually those issues arise if a BufferedWriter is used under the hood which is not closed, maybe the original response writer is wrapped in a filter here and the filter forgets to properly close the BuffetexWriter?

**Comment by YegorKozlov** [comment]
> Usually those issues arise if a BufferedWriter is used under the hood which is not closed, maybe the original response writer is wrapped in a filter here and the filter forgets to properly close the BuffetexWriter?

I don't see any filters that could do that. If explicit close() can cause issues then we can try a different strategy: instead of passing response.getWriter() to Gson we can first serialize the entire payload into a string and then pass that string to response.getWriter() . That will work too. 

What puzzles me is why it works on local and when connected to remote IP and does not work when connected via host name. I  suspect it could be HTTP/1.1 vs HTTP2  .

File: bundle/src/test/java/com/adobe/acs/commons/mcp/impl/ControlledProcessManagerServletTest.java [hunk 1]
```diff
@@ -164,6 +164,7 @@ public void test2749() throws IOException, ServletException {
 
         servlet.doGet(request, response);
 
+        writer.close(); // the servlet engine closes the response stream
```
Comment (kwin) [inline bundle/src/test/java/com/adobe/acs/commons/mcp/impl/ControlledProcessManagerServletTest.java:167]: try with resources on bos and writer instead?

**Comment by davidjgonzalez** [comment]
Ill add changlog and update test fo try-resources

**Pull request merged**